### ISSUE TITLE
Accessibility Statement should be its own document

### ISSUE THREAD

**Issue by mgifford** [usds/ditap-curriculum-update#639]
It shouldn't be bundled in https://github.com/usds/ditap-curriculum-update/blob/main/1_About/1A_What-Is-DITAP/1A.1_About-DITAP.md
